_posts/2022-01-31-changelog-164.markdown

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #164"
+date: 2022-01-31 13:00:00 +0300
+---
+
+This is the last release that supports the 2021.2 IDEs. To receive plugin updates further, please upgrade
+your IDE to 2021.3.
+
+## New Features
+
+* [#8441] Support [`inline_const`](https://doc.rust-lang.org/nightly/unstable-book/language-features/inline-const.html) and [`inline_const_pat`](https://doc.rust-lang.org/nightly/unstable-book/language-features/inline-const-pat.html) features
```

Comment:
Let's be more precise here. As far as I know now the plugin can parse the corresponding syntax and annotate it as experimental